examples/Hermitian_Phasing.py

```diff
@@ -67,10 +52,13 @@
         fl.next("Zeroth order phasing correction applied")
         fl.image(data)
         #{{{ Applying the phase corrections
+        fl.basename = "(Hermitian diagnostic)"
         best_shift, max_shift = hermitian_function_test(
             select_pathway(data.C.mean(indirect), signal_pathway),fl=fl)
+        fl.basename = None
```

Comment:
I use the basename to clearly label the diagnostic plots.

note that the xlims of the diagnostic plot are chosen very weirdly.

Reply:
I'm sorry for needing an explanation I'm still pretty hand wavy with what the hermitian function test is doing other than centering the echo at 0. I'm assuming you are talking about the line with: `xlim(np.array(rel_shift)*max_shift/1e-3)` and I think that is using the relative shift that we input and goes to the max shift so what is weird about this?

Reply:
I was just looking at the plot.  I think you just don't want to set the xlims, so that it does the thing where the data automatically fills the window.

Reply:
okay so I think by getting rid of the xlims it's fixed?